src/north/cli/goldstone/north/cli/portchannel.py

```diff
@@ -120,19 +121,31 @@ def show(session, id=None):
 
     rows = []
     for item in items:
-        rows.append(
-            [
-                item["portchannel-id"],
-                item["state"]["oper-status"].lower(),
-                item["state"]["admin-status"].lower(),
-                ", ".join(natsorted(list(item["state"].get("interface", [])))),
-            ]
-        )
+        if "config" in item and "mode" in item["config"]:
```

Comment:
Could you tell me why this `if`, `elif` is needed?

Why we can't use 

```python
item["state"]["admin-status"].lower(),
", ".join(natsorted(list(item["state"].get("interface", [])))),
```

when `mode` doesn't exist in `item["config"]`?

Reply:
This change has been addressed in the latest commit. `if` and `elif` have been removed and we check the presence of each state key before we display them.